src/components/buttons/buttons.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <router-link :to="{ name: 'Result', params: { name: rpayDuration.name } }">
+      <button
+        class="
+          w-full
+          focus:outline-none
+          font-semibold
+          lg:text-lg
+          text-base
+          rounded-lg
+          px-5
+          py-3
+          bg-white
+          group-focus:text-yellow-300
+        "
+        :class="styleActiveRoute"
+        @click="getResultMobile(2, 20)"
+      >
+        {{ rpayDuration.name }}
+      </button>
+    </router-link>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    getResultMobile: {
+      type: Function,
+      required: true,
+    },
+    rpayDuration: {
+      type: Object,
+    },
+  },
+  computed: {
+    styleActiveRoute() {
+      let color = "";
```

Comment:
You don't need a new variable here, rewrite like this
`
  if (this.$route.params.name == this.rpayDuration?.name) {
         return "bg-red-700 text-white ";
   }
   return color = ""
   `

Reply:
done